src/test/java/org/jenkinsci/plugins/ownership/model/workflow/OwnershipGlobalVariableTest.java

```diff
@@ -0,0 +1,166 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.jenkinsci.plugins.ownership.model.workflow;
+
+import com.synopsys.arc.jenkins.plugins.ownership.OwnershipDescription;
+import com.synopsys.arc.jenkins.plugins.ownership.jobs.JobOwnerHelper;
+import com.synopsys.arc.jenkins.plugins.ownership.nodes.NodeOwnerHelper;
+import hudson.model.Action;
+import hudson.model.Result;
+import hudson.model.labels.LabelAtom;
+import hudson.model.queue.QueueTaskFuture;
+import hudson.slaves.DumbSlave;
+import hudson.tasks.MailAddressResolver;
+import hudson.tasks.Mailer;
+import java.util.Arrays;
+import javax.annotation.Nonnull;
+import jenkins.model.JenkinsLocationConfiguration;
+import static org.hamcrest.Matchers.*;
+import org.jenkinsci.plugins.workflow.cps.CpsFlowDefinition;
+import org.jenkinsci.plugins.workflow.job.WorkflowJob;
+import org.jenkinsci.plugins.workflow.job.WorkflowRun;
+import static org.junit.Assert.assertThat;
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.jvnet.hudson.test.JenkinsRule;
+import org.powermock.api.mockito.PowerMockito;
+import org.powermock.core.classloader.annotations.PrepareForTest;
+import org.powermock.modules.junit4.PowerMockRunner;
+
+/**
+ * Tests of {@link OwnershipGlobalVariable}.
+ *
+ * @author Oleg Nenashev
+ */
+@RunWith(PowerMockRunner.class)
+@PrepareForTest({MailAddressResolver.class, Mailer.class, Mailer.DescriptorImpl.class})
+public class OwnershipGlobalVariableTest {
+
+    @Rule
+    public JenkinsRule j = new JenkinsRule();
+
+    private Mailer.DescriptorImpl descriptor;
+
+    //TODO: We have to do Mocking due to the Bad Mailer Plugin API
+    @Before
+    public void setupMailOptions() throws Exception {
+        PowerMockito.spy(Mailer.class);
+        descriptor = PowerMockito.mock(Mailer.DescriptorImpl.class);
+        PowerMockito.doReturn(descriptor).when(Mailer.class, "descriptor");
+        PowerMockito.when(descriptor.getDefaultSuffix()).thenReturn("mailsuff.ix");
```

Comment:
Not really sure what you are doing here but did you mean to add a test dep on `org.jvnet.mock-javamail:mock-javamail`?